README.md

```diff
@@ -8,7 +8,16 @@ As (not really) specified in https://github.com/MohGovIL/Ramzor
 
 A pythonic implementation of the verification process can be found in [`verify.py`](verify.py).
 
-It requires the [`cryptography`](https://pypi.org/project/cryptography/) package to be installed.
+Run the following command to install the dependency packages:
+```bash
+pip install -r requirements.txt
```

Comment:
I recommend upgrading the entire flow to use `pipenv` which is a slightly more modern approach

Reply:
Fixed